setup.py

```diff
@@ -15,6 +15,25 @@
 from pathlib import Path
 from setuptools import setup, find_packages
 
+import os
+if os.name == 'nt':
+    try:
+        import fiona
+    except:
+        print(
+            ">>>>>> Packages fiona and gdal have a known install issue on windows. <<<<<<"
+        )
+        print(
+            ">>>>>>>>>>> Please install fiona before installing qiskit-metal. <<<<<<<<<<<"
+        )
+        print(
+            ">>>>>>> For detailed instructions refer to the qiskit-metal FAQ page <<<<<<<"
+        )
+        print(
+            ">>>>>>>>>>>>> https://qiskit.org/documentation/metal/faq.html <<<<<<<<<<<<<<"
+        )
+        raise
```

Comment:
I do not understand, why is it giving you that? `raise` just re-raises whatever exception caused entering in the except block. It works just fine on my end and the error states: `ModuleNotFoundError: No module named 'fiona'`. Is this just a weird-thing happening on your end? Or is this a OS-specific behavior? If OS-specific I can modify it to over-rule it.

Reply:
Oh, sorry, that was a weird thing on my end. I had fiona installed, so I just put a `raise` into the try block to see what the error message looked like.

The ModuleNotFoundError is fine. You could raise a custom error to emphasize that the installation is being stopped intentionally rather than as a side effect of bad import in setup.py but it is probably clear enough with the extra text.